hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -266,12 +267,15 @@ public String showCommitPartitions(
 
     HoodieActiveTimeline activeTimeline = HoodieCLI.getTableMetaClient().getActiveTimeline();
     HoodieTimeline timeline = activeTimeline.getCommitsTimeline().filterCompletedInstants();
-    HoodieInstant commitInstant = new HoodieInstant(false, HoodieTimeline.COMMIT_ACTION, instantTime);
 
-    if (!timeline.containsInstant(commitInstant)) {
+    Option<HoodieInstant> hoodieInstantOptional = getCommitOrReplaceCommitInstant(timeline, instantTime);
+    if (!hoodieInstantOptional.isPresent()) {
       return "Commit " + instantTime + " not found in Commits " + timeline;
     }
-    HoodieCommitMetadata meta = HoodieCommitMetadata.fromBytes(activeTimeline.getInstantDetails(commitInstant).get(),
+
+    HoodieInstant hoodieInstant = hoodieInstantOptional.get();
+
+    HoodieCommitMetadata meta = HoodieCommitMetadata.fromBytes(activeTimeline.getInstantDetails(hoodieInstant).get(),
         HoodieCommitMetadata.class);
     List<Comparable[]> rows = new ArrayList<>();
     for (Map.Entry<String, List<HoodieWriteStat>> entry : meta.getPartitionToWriteStats().entrySet()) {
```

Comment:
it'd be nice to  compute totalfFilesReplaced and show it in the table. It could be 0 for regular commits.

Reply:
This I will pick up in next PR along with showing extraMetadata.

Reply:
sounds good. Its important for debugging to at least show commit action type (commit vs deltacommit vs replacecommit) in the output. If possible, add that information now. If not, please ping me when you have next PR.